addon/components/copy-button.js

```diff
@@ -60,14 +60,15 @@ export default class CopyButtonComponent extends Component {
    * @returns {Object} newly created ClipboardJS object
    */
   _createClipboard() {
-    const { clipboardText: text, delegateClickEvent } = this;
+    const { clipboardText: text, containerId, delegateClickEvent } = this;
     const trigger =
       delegateClickEvent === false
         ? this._buttonElement
         : `#${this._buttonElement.id}`;
 
     return new ClipboardJS(trigger, {
       text: typeof text === 'function' ? text : undefined,
+      container: document.getElementById(containerId) || document.body,
```

Comment:
Do you know if the following works? I think we should let the internal library decide it's default container (it's body currently but they may change that)
```suggestion
      container: containerId ? document.getElementById(containerId) : null,
```

Reply:
I tried it and it seems to work, but you're right, we should let the internal library decide what should be the default container. However, I'm not sure we should send `null` otherwise, because in [clipboardJS](https://github.com/zenorocha/clipboard.js/blob/master/src/clipboard.js#L30):
```js
this.container = (typeof options.container === 'object')   ? options.container : document.body;
```
and `typeof null` is `object`. WDYT? :thinking:

Reply:
We can use `undefined` the same way as https://github.com/qonto/ember-cli-clipboard/pull/1/files#diff-6de189e166f4519ad6799386b68427c5R70

Reply:
I updated the line. :)

Reply:
Cool, thanks!